### ISSUE TITLE
Different "MinGameTime" in D2BotLead.dbj

### ISSUE THREAD

**Issue by PoBErs** [kolton/d2bot-with-kolbot#833]
Is it possible for different leaders to have different minimum game time set by the D2BotLead? I know you can configure minimum and maximum time in the character config but I'd like to have the minimum game set by the D2BotLead incase of crash. But one bot is solo so I don't want to limit that one for a long game time while the others are a party. So if the leader drops I don't want him to create a new game too fast.

**Comment by PoBErs** [comment]
Yeah I have been thinking about the options you described. But I rather just have 1 d2bot client. 
I tried to do what you said on the first sentence, about making a new D2BotLead.dbj and renaming it like D2BotLead2.dbj, but yeah as expected it kept crashing.

There is a possibility to override the channel join in both D2BotLead and D2BotChannel and I am using it for other reasons, but mabye there is a possibility to also override the minimum game time there? I tried with no success, but then I am a total novice in programming.

So if anyone knows the "var ChannelConfig" override that would be awesome. Thank you!

**Comment by DarkHorseDre** [comment]
Yeah the issue with the lead entry is its referenced in the .exe and .dll (I grep'd all files).

as for the channeljoin - no idea.  issue is you need 2 leaders and channeljoin is for a non leader to join other games.. if you have one leader using lead entry, what will the other use to lead their runs?

I hope someone else can help ya!

**Comment by mf022** [comment]
you could think to use the staggered version of D2BotLead. check https://github.com/mf022/d2bs/wiki/Misc_options#staggered-delays-for-creating-games

anyway, the starter scripts can be copied with different names, but don't include digits in those new names, but only letters. No other files/folders need to be cloned/copied.

by example, you can have a version or D2BotLeadSP with 0 values for all game timers, because it's about local created games.
